cmd/kepler/main.go

```diff
@@ -27,46 +29,29 @@ func main() {
 	logVersionInfo(logger)
 	printConfigInfo(logger, cfg)
 
-	var g run.Group
+	logger.Info("Staring Kepler services")
 
-	logger.Info("Starting Kepler...")
+	// create & register all services with run group
+	services := createServices(logger)
 	ctx, cancel := context.WithCancel(context.Background())
-	{
-		g.Add(waitForInterrupt(ctx, os.Interrupt))
-	}
-
-	{
-		// TODO: replace with monitor.Start()
-		g.Add(
-			func() error {
-				logger.Info("Monitor is running. Press Ctrl+C to stop.")
-				<-ctx.Done()
-				logger.Info("Monitor is done running.")
-				return nil
-			},
-			func(err error) {
-				logger.Warn("Shutting down...:", "error", err)
-				cancel()
-			},
-		)
-	}
-
-	{
-		// TODO: replace with server.Start()
+	var g run.Group
+	for _, s := range services {
 		g.Add(
 			func() error {
-				logger.Info("HTTP server is running. Press Ctrl+C to stop.")
-				<-ctx.Done()
-				return nil
+				return s.Start(ctx)
 			},
 			func(err error) {
-				logger.Info("HTTP Server: Shutting down...:", "error", err)
+				if cleanupErr := s.Stop(); cleanupErr != nil {
+					logger.Warn("service terminated with error: %v", "service", s.Name(), "error", err)
```

Comment:
%v for error is redundant

Reply:
Also we should log both errors separately.